### ISSUE TITLE
No Sound on iOS

### ISSUE THREAD

**Issue by 12yanogden** [dlutton/flutter_tts#458]
## 🐛 Bug Report

No build or runtime errors. The volume is turned up on my simulator and computer. There is simply no sound.

### Expected behavior

That there is sound.

### Reproduction steps

1. Clone my [repo](https://github.com/12yanogden/tts_poc).
2. Run the app on an iOS simulator.
3. Type any word or phrase into the input field.
4. Press "Start Text To Speech".
5. Listen to silence.

### Configuration

**Version:** ^3.8.5

**Platform:** 
- [x] :iphone: iOS

Running on MacOS M1.

**Comment by knight0zh** [comment]
me too

**Comment by yishaiSilver** [comment]
#311 Solved it for me (iPhone SE)

**Comment by 12yanogden** [comment]
Is 3.8.5 not even a valid version then? We always have to downgrade?

**Comment by TranTuanManh** [comment]
Try the below code:

<img width="777" alt="image" src="https://github.com/dlutton/flutter_tts/assets/29418461/11aef6a3-23ce-4c69-ac99-a015016dc110">

**Comment by 12yanogden** [comment]
> Try the below code:
> 
> <img alt="image" width="777" src="https://private-user-images.githubusercontent.com/29418461/301865050-11aef6a3-23ce-4c69-ac99-a015016dc110.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgxOTEyNTQsIm5iZiI6MTcwODE5MDk1NCwicGF0aCI6Ii8yOTQxODQ2MS8zMDE4NjUwNTAtMTFhZWY2YTMtMjNjZS00YzY5LWFjOTktYTAxNTAxNmRjMTEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE3VDE3MjkxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5ZDIzMDYwNzI0YWUzNWMxNTE4MzcwYjNiN2NkNzBjOTdjMTBlY2VhMzA1MGU2MmZlMWEzYjJiNDFiNzA1MmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6F-S16BUdHSk2k8McHaA7N4iUtRR9A19IuUpu-SIGCw">

Thanks for the suggestion. Unfortunately, the issue still persists.

To be clear, I can't get any sound to play in Google Chrome either. I'm not deploying to Chrome, so I don't mind that. The solution might very well not be iOS-specific though.

**Comment by garry-t** [comment]
same for android.

**Comment by rennylangat** [comment]
@12yanogden Did you find a work around for the iOS simulator issue?

**Comment by 12yanogden** [comment]
> @12yanogden Did you find a work around for the iOS simulator issue?

Nope. When I revisit my project, I'm probably going to choose another plugin. This one is simply broken, even for the most basic use case.

**Comment by dlutton** [comment]
@rennylangat I apologize as I have been extremely busy, but will look into some of these issues in the next couple of days.  Is this only occurring in your simulator or on an actual device?

**Comment by rennylangat** [comment]
@dlutton It happens on both.
On a physical device I get this `Unable to list voice folder` error message upon running. Could be an issue with AV Speech Synthesizer?

**Comment by mijiga** [comment]
Adding this after initialisation fixed it for me:

let flutterTts = FlutterTts();
await flutterTts.setSharedInstance(true);
await flutterTts.setIosAudioCategory(IosTextToSpeechAudioCategory.playback,
    [
      IosTextToSpeechAudioCategoryOptions.allowBluetooth,
      IosTextToSpeechAudioCategoryOptions.allowBluetoothA2DP,
      IosTextToSpeechAudioCategoryOptions.mixWithOthers,
      IosTextToSpeechAudioCategoryOptions.defaultToSpeaker
    ],
    IosTextToSpeechAudioMode.defaultMode
);

**Comment by hvlamdev** [comment]
I also encountered a situation where there was no sound, but in my case it was because I set it to silent mode (Silent switch).
I used `IosTextToSpeechAudioCategory.playback` to be able to play sound even in silent mode.
